server/channels/app/plugin_api_test.go

```diff
@@ -2365,3 +2365,44 @@ func TestPluginServeMetrics(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, "METRICS SUBPATH", string(body))
 }
+
+func TestPluginUpdateChannelMembersNotifications(t *testing.T) {
```

Comment:
This doesn't check for the ws event. Might be good to check for that in an app/api4 layer test. 0/5

Reply:
It might also be nice to structure this as a "real plugin test", e.g.

https://github.com/mattermost/mattermost/blob/0496483a9ebbe281fbbb560874ecd2440b29442c/server/channels/app/plugin_api_test.go#L136-L164

Previous PRs have found that the API is unit tested but something went wrong when it actually got used over RPC (e.g. gob export missing, or forgot to extend plugin interface). I don't think there's anything wrong her, per se, but having a "real test" would boost that confidence now and going forward.

Reply:
Will do both of those.

For the WS event, I'll add a test for that, but I'll put that in the app layer since it's not strictly plugin-related.

For the real plugin test, I'll probably add that alongside the existing one then. It'd be nice if we could trust the RPC and interface somehow so that we wouldn't need to do that, but I understand why that's probably easier said than done